numba/np/random/random_methods.py

```diff
@@ -0,0 +1,355 @@
+import numpy as np
+
+from numba.core.extending import register_jitable
+
+from numba.np.random._constants import (UINT32_MAX, UINT64_MAX,
+                                        UINT16_MAX, UINT8_MAX)
+from numba.np.random.generator_core import next_uint32, next_uint64
+
+# All following implementations are direct translations from:
+# https://github.com/numpy/numpy/blob/7cfef93c77599bd387ecc6a15d186c5a46024dac/numpy/random/src/distributions/distributions.c
+
```

Comment:
General comment on this, if possible, please add a short docstring to each function to make it easier to understand the code.

Also, some of the comments were presumably ported over from C and still have the c-style comment syntax `/*` and `*/` in the Python comments. You can probably remove those syntax symbols.

Reply:
Added a few docstrings and fixed the comments